src/graphics/sprite.js

```diff
@@ -205,10 +201,20 @@ Crafty.c("Sprite", {
     init: function () {
         this.__trim = [0, 0, 0, 0];
         this.bind("Draw", this._drawSprite);
+        this.bind("LayerAttached", this._setupSpriteImage);
```

Comment:
shouldn't `_setupSpriteImage` be called if _drawLayer already exists here in addition?